README.md

```diff
@@ -68,11 +68,39 @@ The following simple HTML code illustrates the way to embed library view into an
 
             let libView = new LibraryEntryPoint.LibraryView(configuration);
 
-            // TODO: The callback when a library item is clicked on has not been 
-            // provided yet, it is scheduled to be done in the near future.
+            // Register callback functions here.
         </script>
 
     </body>
 </html>
```

Comment:
Thanks for the write-up, @yeexinc. Unfortunately none of the following will be required, as a consumer of `librarie.js`, one will not be concerned about how events are raised -- they only care about how the events can be registered. Please remove the changes below in `ReadMe.md`.